src/app/exercise/exercise.page/exercise.page.ts

```diff
@@ -41,23 +41,30 @@ export class ExercisePage {
   }
 
   onAnswerClick(answerConfig: AnswerConfig<string>): void {
+    
     if (this.isQuestionCompleted) {
       this.state.playAnswer(answerConfig);
       return;
     }
     const answer: string | null = answerConfig.answer;
+    
     if (!answer) {
       throw new Error(`Clicked answer is ${answer}`)
     }
     const isRight: boolean = this.state.answer(answer);
-    if (isRight) {
-      this.rightAnswer = answer;
+
+    if (isRight || true) {
+      if(isRight){
+        this.rightAnswer = answer;
+      } else {
+        this.wrongAnswers.push(answer);                
+      }     
       setTimeout(() => {
         this.rightAnswer = null;
+        this.wrongAnswers = [];
```

Comment:
This should not be inside the setTimeout.
The set timeout goal is to flash the right answer in green, and then fade the green out. I would like the wrong answers indication (Red) to be removed immediately upon pressing, so there is no need for delay.